Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jupyter/base-notebook
+
+USER root
+RUN apt-get update -y && apt-get install -y --no-install-recommends build-essential graphviz
+
```

Comment:
If we want to execute hello.py from [this page](https://graphviz.readthedocs.io/en/stable/examples.html).
We have to install xdg-open.
So, I add this line here. `RUN apt-get install -y xdg-utils --fix-missing`
And, I checked hello.py works.

Reply:
xdg-open is package for GUI (x11).
i would rather not  install it. 

may be it used `g.view()` in hello.py.